game/src/game/player/player_behaviour.rs

```diff
@@ -1258,7 +1272,38 @@ fn player_attack(
                         ))
                         .id()
                 },
-                PlayerWeapon::Sword => {
+                PlayerCombatStyle::Melee => {
+                    let (
+                        damage,
+                        self_pushback,
+                        self_pushback_ticks,
+                        pushback,
+                        pushback_ticks,
+                    ) = match *melee_weapon {
+                        PlayerMeleeWeapon::Hammer => (
+                            config.hammer_attack_damage,
+                            config.hammer_self_pushback,
+                            config.hammer_self_pushback_ticks,
+                            config.hammer_pushback,
+                            config.hammer_pushback_ticks,
+                        ),
+                        PlayerMeleeWeapon::Sword => (
+                            config.sword_attack_damage,
+                            config.sword_self_pushback,
+                            config.sword_self_pushback_ticks,
+                            config.sword_pushback,
+                            config.sword_pushback_ticks,
+                        ),
+                    };
+
+                    // Slow the player down when they attack with the Hammer
+                    if let PlayerMeleeWeapon::Hammer = *melee_weapon {
+                        player_stats.set(
+                            StatType::MoveVelMax,
+                            config.hammer_max_move_vel,
+                        );
```

Comment:
@odecay please check how I'm setting a slower speed stat for the player here (to confirm that it won't interfere with the passives)...